107/src/test/java/org/ehcache/ParsesConfigExtensionTest.java

```diff
@@ -70,30 +63,9 @@ public void testConfigParse() throws ClassNotFoundException, SAXException, Insta
     assertThat(jsr107Service.getDefaultTemplate(), equalTo("tinyCache"));
     assertThat(jsr107Service.getTemplateNameForCache("foos"), equalTo("stringCache"));
     assertThat(jsr107Service.getTemplateNameForCache("bars"), nullValue());
-
-    // TEST ENDS HERE, BELOW IS AN EXAMPLE OF CODE FOR OUR 107 CACHE MANAGER
-    // IT WOULD BUILD OUR 107 CACHE MANAGER INSTANCE AS ABOVE, KEEPING A REF TO THE DefaultJsr107Service ABOVE
-
-    // random javax.cache.CacheManager.createCache(name, cfg) example:
-    String name = "bar";
-    CompleteConfiguration cfg = new MutableConfiguration();
-
-    CacheConfigurationBuilder configurationBuilder = configuration
-        .newCacheConfigurationBuilderFromTemplate(jsr107Service.getDefaultTemplate(), cfg.getKeyType(), cfg.getValueType());
-    final String template = jsr107Service.getTemplateNameForCache(name);
-    if(template != null) {
-      configurationBuilder = configuration
-          .newCacheConfigurationBuilderFromTemplate(name, cfg.getKeyType(), cfg.getValueType());
-    }
-    if(configurationBuilder != null) {
-      configurationBuilder = CacheConfigurationBuilder.newCacheConfigurationBuilder();
-    }
-    // set all attributes from cfg, to complete or override
-    // and finally build our CacheConfig:
-    final CacheConfiguration cacheConfiguration = configurationBuilder.buildConfig(cfg.getKeyType(), cfg.getValueType());
-    // final Cache cache = cacheManager.createCache(name, cacheConfiguration); // throws NPE because of no Store.Provider Service present
   }
```

Comment:
yeah, and it wasn't really intended to be part of the test. It was just a convenient place for alex to leave the usage code for me to use later.